tensorflow/core/kernels/data/dataset_test_base.h

```diff
@@ -38,9 +38,35 @@ limitations under the License.
 #include "tensorflow/core/platform/types.h"
 #include "tensorflow/core/util/ptr_util.h"
 
+#include "tensorflow/core/lib/io/zlib_compression_options.h"
+#include "tensorflow/core/lib/io/zlib_outputbuffer.h"
+#include "tensorflow/core/platform/env.h"
+
 namespace tensorflow {
 namespace data {
 
+enum CompressionType : int { ZLIB = 0, GZIP = 1, RAW = 2, UNCOMPRESSED = 3 };
```

Comment:
if you use `enum class` you will not need to worry about unsupported enum values.

Reply:
Good suggestion! Changed it.